samples/emoji-search/android/src/main/java/app/cash/zipline/samples/emojisearch/EmojiSearchActivity.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 Square, Inc.
+ * Copyright (C) 2022 Block, Inc.
```

Comment:
I was always told to never change these dates.

As to changing the copyright holder name, I would recommend consulting our open source legal people to ensure this is valid to do.

Reply:
More specifically:
* Don’t change these dates ’cause it’s not useful.
* Do use '2022 Block, Inc' in new code

Reply:
Reverted everything and will stick with `current year Block, Inc` on new files